From ce807219123614d8496b29b591b8147dc190db54 Mon Sep 17 00:00:00 2001
From: Avesh Agarwal <avagarwa@redhat.com>
Date: Tue, 10 Dec 2013 13:38:23 -0500
Subject: [PATCH 01/20] NM-openswan was missing support for
 nm-openswan-auth-dialog.desktop.in.in. So added a new
 nm-openswan-auth-dialog.desktop.in.in, and modified related Makefile and
 configure.ac files.

---
 auth-dialog/Makefile.am                           | 15 ++++++++++++++-
 auth-dialog/nm-openswan-auth-dialog.desktop.in.in |  7 +++++++
 configure.ac                                      |  1 +
 3 files changed, 22 insertions(+), 1 deletion(-)
 create mode 100644 auth-dialog/nm-openswan-auth-dialog.desktop.in.in

diff --git a/auth-dialog/Makefile.am b/auth-dialog/Makefile.am
index 4b5a01d..d1e8711 100644
--- a/auth-dialog/Makefile.am
+++ b/auth-dialog/Makefile.am
@@ -22,5 +22,18 @@ nm_openswan_auth_dialog_LDADD = \
 	$(GTK_LIBS) \
 	$(GNOMEKEYRING_LIBS)
 
-CLEANFILES = *~
+@INTLTOOL_DESKTOP_RULE@
+desktopdir = $(datadir)/applications
+desktop_in_in_files = nm-openswan-auth-dialog.desktop.in.in
+desktop_in_files = nm-openswan-auth-dialog.desktop.in
+desktop_DATA = $(desktop_in_files:.desktop.in=.desktop)
 
+EXTRA_DIST = \
+	$(desktop_in_in_files)
+
+CLEANFILES = \
+	*~ \
+	$(desktop_DATA)
+
+DISTCLEANFILES = \
+	$(desktop_in_files)
diff --git a/auth-dialog/nm-openswan-auth-dialog.desktop.in.in b/auth-dialog/nm-openswan-auth-dialog.desktop.in.in
new file mode 100644
index 0000000..9c91b6b
--- /dev/null
+++ b/auth-dialog/nm-openswan-auth-dialog.desktop.in.in
@@ -0,0 +1,7 @@
+[Desktop Entry]
+_Name=Network
+_Comment=Request VPN authentication
+Exec=@LIBEXECDIR@/nm-openswan-auth-dialog
+Terminal=false
+Type=Application
+NoDisplay=true
diff --git a/configure.ac b/configure.ac
index b214ef7..10af655 100644
--- a/configure.ac
+++ b/configure.ac
@@ -114,6 +114,7 @@ AC_CONFIG_FILES([
 Makefile
 src/Makefile
 auth-dialog/Makefile
+auth-dialog/nm-openswan-auth-dialog.desktop.in
 properties/Makefile
 po/Makefile.in
 ])
-- 
1.8.3.1

